htsinfer/motif_enrichment.py

```diff
@@ -0,0 +1,100 @@
+"""Calculate motif enrichment from sample data."""
+
+import numpy as np
+from scipy.stats.distributions import binom
+from typing import (Dict, List)
+
+FOREGROUND_DICT = {"UGAUUC": 5, "UAAACC": 3, "AAGUUACCU": 1,
+                   "AAGCCUU": 1, "AGUUCUA": 1, "UUUCCCG": 5}
+BACKGROUND_DICT = {"UGAUUC": 3, "UAAACC": 5, "AAGCCUUAU": 1,
+                   "AGUUCUA": 1, "UUUCCCG": 5, "UUGGAA": 7}
+
+
+def motif_enrichment(
+    foreground: Dict[str, int],
+    background: Dict[str, int]) -> Dict: 
+    """Calculates enrichment and p-values of motifs with similar lengths.
+
+    Args:
+        foreground: dictionary of motifs with counts
+        background: dictionary of motifs with counts
+
+    Returns:
+        Dictionary including motifs, sum of counts of motifs with
+        similar lenght, enrichment score and p-value
+
+    Example:
+        foreground = {"TGATTC": 5, "TAAACC": 3, "AAGTTACCT": 1,
+        "AAGCCTT": 1, "AGTTCTA": 1, "TTTCCCG": 5}
+        background = {"TGATTC": 3, "TAAACC": 5, "AAGCCTTAT": 1,
+        "AGTTCTA": 1, "TTTCCCG": 5, "TTGGAA": 7}
+        >>> motif_enrichment(foreground, background)
```

Comment:
The return value in the example is not consistent with the description above (there is no sum of counts returned).